wzdx/tools/date_tools.py

```diff
@@ -32,11 +41,29 @@ def parse_datetime_from_unix(time):
 
     if type(time) == str:
         try:
-            return datetime.fromtimestamp(float(time), tz=timezone.utc)
+            return datetime_from_unix(float(time))
         except ValueError:
             return None
     elif type(time) == int or type(time) == float:
+        return datetime_from_unix(time)
+
+
+def datetime_from_unix(time):
+    try:
+        # Fails for milliseconds
```

Comment:
Does this happen often? Do we have a case to pull out a separate function for parsing when we know it's milliseconds?

Reply:
I just wanted a function where I didn't need to care the exact format (seconds or milliseconds), more convenient that way

Reply:
Is there a check we can add in here like if time > xxx it's coming in ms so use time/1000? It's best not to rely on an exception if possible, they are much more expensive than a check beforehand.

Reply:
I found the number lol, turns our 32536850399 succeeds, but 32536850400 overflows it. Weird. I not just check if it is larger than this instead of using a try catch

Reply:
its not even a clean power of 2 (or off by 1), wtf